frontend/test/metabase/scenarios/joins/reproductions/20519-cannot-join-on-aggregation-with-implicit-joins-and-nested-query.cy.spec.js

```diff
@@ -0,0 +1,72 @@
+import {
+  restore,
+  enterCustomColumnDetails,
+  visualize,
+} from "__support__/e2e/cypress";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { ORDERS, ORDERS_ID, PRODUCTS } = SAMPLE_DATABASE;
+
+const questionDetails = {
+  name: "20519",
+  query: {
+    "source-query": {
+      "source-table": ORDERS_ID,
+      aggregation: [["count"]],
+      breakout: [
+        ["field", PRODUCTS.CATEGORY, { "source-field": ORDERS.PRODUCT_ID }],
+      ],
+    },
+    joins: [
+      {
+        fields: "all",
+        "source-table": 1,
```

Comment:
Which table is it?
```suggestion
        "source-table": PRODUCTS_ID,
```

Reply:
Oh, good catch. I overlooked it because `1` is usually the sample database.